lib/linguist/languages.yml

```diff
@@ -641,9 +641,28 @@ Bluespec:
   color: "#12223c"
   extensions:
   - ".bsv"
+  aliases:
+  - bluespec bsv
+  - bsv
   tm_scope: source.bsv
   ace_mode: verilog
+  codemirror_mode: verilog
+  codemirror_mime_type: text/x-systemverilog
   language_id: 36
+Bluespec BH:
+  type: programming
+  group: Bluespec
+  color: "#12223c"
+  extensions:
+  - ".bs"
+  aliases:
+  - bh
+  - bluespec classic
+  tm_scope: source.haskell
+  ace_mode: haskell
+  codemirror_mode: haskell
+  codemirror_mime_type: text/x-haskell
+  language_id: 626c75650
```

Comment:
Did you run the script to generate the `language_id`? Unless the ids were in hexadecimal all along and I didn't know, there  shouldn't be a "c" in there.

Reply:
I'm returning to these changes after a long pause, so I don't recall.  I'll run the script to generate a new ID and push another commit for it.  Thanks!